routes/road-properties.js

```diff
@@ -18,29 +18,77 @@ function getHandler (req, res) {
     )));
 }
 
-function deleteHandler (req, res) {
-  // This method will have issues if commas are used in road IDs
-  const idsToDelete = req.query.ids && req.query.ids.split(',');
-  if (!idsToDelete) {
-    return res(Boom.badRequest('Must provide a querystring list of road `ids` to delete'));
-  }
-  if (new Set(idsToDelete).size < idsToDelete.length) {
-    return res(Boom.badRequest('Cannot have duplicate road `ids`'));
-  }
-
+function getIdsHandler(req, res) {
   knex('road_properties')
-    .whereIn('id', idsToDelete)
     .select('id')
-    .then(idsInDB => {
-      if (idsInDB.length < idsToDelete.length) {
-        return res(Boom.badRequest(`Some requested road IDs not present in database: ${JSON.stringify(_.difference(idsToDelete, idsInDB))}`));
-      } else {
-        knex('road_properties')
-          .whereIn('id', idsToDelete)
-          .delete()
-          .then(() => res(idsToDelete));
-      }
-    });
+    .map(entry => entry.id)
+  .then(res);
+}
+
+function createHandler(req, res) {
+  knex('road_properties')
+    .insert({
+      id: req.params.road_id,
+      properties: '{}'
+    })
+  .then(function(response) {
+    return res({ id: req.params.road_id }).type('application/json');
+  })
+  .catch(function(err) {
+    if (err.constraint === 'road_properties_pkey') {
+      return res(Boom.conflict());
+    }
+
+    return res(Boom.badImplementation());
+  });
+}
+
+function moveHandler(req, res) {
+  knex('road_properties')
+    .where({ id: req.params.road_id })
+    .update({
+      id: req.payload.id
+    })
+  .then(function(response) {
```

Comment:
Also for moving, the `current_way_tags` values should be updated as well, if any exist. These aren't explicit foreign key relationships, but the OSM schema's `or_vpromms` key value is used as a join key (with `road_properties.id`) for several purposes.

So, for any `current_way.id`, if there is a `current_way_tags.k` of `or_vpromms`, and that row's `current_way_tags.v` is equal to the `road_properties.id` that's being moved, then update the `current_way_tags.v` value.

(Let's not worry about handling `current_way_tags.v` alteration for the `road_properties.id` delete handler.)